### PR TITLE
feat: create come-si-realizza-design-servizi-pubblici

### PR DESCRIPTION
**Pull request by Francescopog**
## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> create 2022-02-22-come-si-realizza-design-servizi-pubblici
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4d8a62d478c7cc9bf5e4f39a128c04cc72cb80bd
</p>

File: _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Durante gli ultimi 20 minuti della Community call, come sempre, i partecipanti hanno avuto uno spazio a disposizione per domande e curiosità.
+
+Il tema del *design system* sarà protagonista nei mesi di marzo e aprile di due **community lab**, veri e propri laboratori: occasioni ideate per scoprire e imparare insieme gli strumenti utili a collaborare per migliorare i servizi pubblici digitali. 
+
+
+**[Guarda il video completo della Community call](https://www.youtube.com/watch?v=Q6bCRei3UHU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL)** sul canale YouTube del Dipartimento per la trasformazione digitale e scarica la presentazione.
+
+Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor) e [Designers Italia](https://twitter.com/developersITA?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor).
```
Comment (bfabio) [inline _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md:28]:
```suggestion
Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/developersITA).
```

File: _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+subtitle: Pubblicato il video della Community call del 9 febbraio sull’importanza del design system
+date: 2022-02-22 12:00:00 +0200
+tags: developers designers designsystem
+categories: news
+author: Michela Gabrieli
+image: /assets/icons/logo-it.png
+locale: it
+---
+
+Grande successo per la Community call dello **scorso 9 febbraio**, dedicata al *design system* per i servizi pubblici digitali, con oltre **150 partecipanti**. Abbiamo parlato dell’importanza del *design system** per creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. Per farlo ci siamo affidati ad alcuni dei protagonisti di questo processo: Daniele Tabellini, Marco Liberati e Giacomo Grassi.
```
Comment (bfabio) [inline _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md:13]:
```suggestion
Grande successo per la Community call dello **scorso 9 febbraio**, dedicata al *design system* per i servizi pubblici digitali, con oltre **150 partecipanti**. Abbiamo parlato dell’importanza del *design system* per creare siti e servizi coerenti, familiari e fruibili da tutti i cittadini. Per farlo ci siamo affidati ad alcuni dei protagonisti di questo processo: Daniele Tabellini, Marco Liberati e Giacomo Grassi.
```

File: _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+**Marco Liberati**, membro attivo della community, è poi intervenuto per raccontare lo stato del Kit React “design-react-kit”, che supporta chi sviluppa nell’implementazione di Bootstrap Italia. Marco ha prima illustrato le basi del kit e ne ha poi raccontato l’evoluzione delle varie versioni fino all’attuale 4.0.
+
+**Giacomo Grassi**, *User Experience & Digital Process Director* presso INPS, ha descritto la sua esperienza di creazione di un design system nel settore pubblico. Ha quindi spiegato le specifiche necessità per un ente pubblico di grande complessità, che offre oltre 450 servizi web. Ha sottolineato l’importanza di creare team di lavoro coesi, valorizzare le competenze interne preesistenti e coinvolgere anche i collaboratori esterni fin dal primo giorno di lavoro.
+
+Durante gli ultimi 20 minuti della Community call, come sempre, i partecipanti hanno avuto uno spazio a disposizione per domande e curiosità.
+
+Il tema del *design system* sarà protagonista nei mesi di marzo e aprile di due **community lab**, veri e propri laboratori: occasioni ideate per scoprire e imparare insieme gli strumenti utili a collaborare per migliorare i servizi pubblici digitali.
+
```
Comment (bfabio) [inline _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md:24]:
```suggestion
```

File: _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+**[Guarda il video completo della Community call](https://www.youtube.com/watch?v=Q6bCRei3UHU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL)** sul canale YouTube del Dipartimento per la trasformazione digitale e [scarica la presentazione](/assets/images/posts/2022-02-22/Importanza-di-un design-system-per-i-servizi-digitali.pdf).
+
+Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/developersITA).
+
+
```
Comment (bfabio) [inline _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md:30]:
```suggestion
```

**Review by bfabio — APPROVED** [review]

File: _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+Durante gli ultimi 20 minuti della Community call, come sempre, i partecipanti hanno avuto uno spazio a disposizione per domande e curiosità.
+
+Il tema del *design system* sarà protagonista nei mesi di marzo e aprile di due **community lab**, veri e propri laboratori: occasioni ideate per scoprire e imparare insieme gli strumenti utili a collaborare per migliorare i servizi pubblici digitali.
+
+**[Guarda il video completo della Community call](https://www.youtube.com/watch?v=Q6bCRei3UHU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL)** sul canale YouTube del Dipartimento per la trasformazione digitale e [scarica la presentazione](/assets/images/posts/2022-02-22/Importanza-di-un design-system-per-i-servizi-digitali.pdf).
+
+Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/developersITA).
```
Comment (bfabio) [inline _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md:27]:
```suggestion
Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/DesignersITA).
```

File: _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+Durante gli ultimi 20 minuti della Community call, come sempre, i partecipanti hanno avuto uno spazio a disposizione per domande e curiosità.
+
+Il tema del *design system* sarà protagonista nei mesi di marzo e aprile di due **community lab**, veri e propri laboratori: occasioni ideate per scoprire e imparare insieme gli strumenti utili a collaborare per migliorare i servizi pubblici digitali.
+
+**[Guarda il video completo della Community call](https://www.youtube.com/watch?v=Q6bCRei3UHU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL)** sul canale YouTube del Dipartimento per la trasformazione digitale e [scarica la presentazione](/assets/images/posts/2022-02-22/Importanza-di-un design-system-per-i-servizi-digitali.pdf).
+
+Per saperne di più **resta in contatto su [Slack](https://app.slack.com/client/T6C27AXE0)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/DesignersITA).
```
Comment (bfabio) [inline _posts/it/2022-02-22-come-si-realizza-design-servizi-pubblici.md:27]:
```suggestion
Per saperne di più **resta in contatto su [Slack](https://slack.developers.italia.it)**, continua a seguirci sui canali Twitter di [Developers Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/DesignersITA).
```

**Pull request merged**